java/src/jmri/jmrix/loconet/messageinterp/MessageInterpretation.properties

```diff
@@ -520,6 +521,25 @@ LN_MSG_OPC_MULTI_SENSE_TRANSP_ZONEH = H
 # replaced by a decimal representation of the LS 4 bits of byte 2 of the message.
 LN_MSG_OPC_MULTI_SENSE_TRANSP_ZONE_UNKNOWN = (Unknown {0})
 
+# Note: in # LN_MSG_OPC_MULTI_SENSE_LONG_TRANSP_REPORT, the following parameter
+# substitutions will be made:
+#   {0} is a string representing the decoder address which triggered the report,
+#       and whether that address is 'long' or 'short', from
+#       LN_MSG_LOCO_ADDRESS_HELPER_MIXED_ADDRESS or
+#       LN_MSG_LOCO_ADDRESS_HELPER_UNMIXED_ADDRESS, as appropriate
+#   {2} is a string representing the direction headed
```

Comment:
{2} shows up twice in the comments.

Reply:
I swear I fixed that a few days ago... At one point I swapped {1} and {2}  to what I thought was a more natural description.  I have updated.